packages/reference/src/resources/MultipleResourceReferenceEditor.tsx

```diff
@@ -34,8 +35,8 @@ type EditorProps = ReferenceEditorProps &
 function ResourceEditor(props: EditorProps) {
   const { setValue, items, apiUrl } = props;
 
-  const onSortStart: SortStartHandler = useCallback((_, event) => event.preventDefault(), []);
-  const onSortEnd: SortEndHandler = useCallback(
+  const onSortStart = useCallback(() => noop(), []);
```

Comment:
Seems a bit of a waste having an empty function in a useCallback, maybe just `noop` is fine